### DISCUSSION TITLE
enable 'legacyTemplate'

### DISCUSSION THREAD

**Discussion by magick93** [storybookjs/addon-svelte-csf#253] [Q&A]
Hello

I've recently upgrade and am getting lots of similar warnings:

```

WARN 🚨 Unable to index files:
WARN - ./src/lib/components/auction/sealed.stories.svelte: Stories file: /home/anton/git/yestech/freedom/packages/ui/src/lib/components/auction/sealed.stories.svelte
WARN is using legacy API.
WARN 
WARN To enable support for it, enable 'legacyTemplate' in "@storybook/addon-svelte-csf" option.

```

But I cant find any docs on where / how to enable the `legacyTemplate`.

Can someone please mention how I do that?

Thanks

**Comment by xeho91** [comment]
https://github.com/storybookjs/addon-svelte-csf/tree/next?tab=readme-ov-file#legacy-api
